packages/ckeditor5-dev-build-tools/tests/plugins/splitCss/splitCss.test.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { join } from 'path';
+import { test } from 'vitest';
+import { rollup, type RollupOutput, type NormalizedOutputOptions } from 'rollup';
+import { verifyDividedStyleSheet } from '../../_utils/utils.js';
+
+import { splitCss } from '../../../src/index.js';
+
+const TEST_BANNER = `/**
+ * License banner.
+ */
+`;
+
+/**
+ * Helper function for creating a bundle that won't be written to the file system.
+ */
+async function generateBundle(
+	input: string,
+	options?: NormalizedOutputOptions,
+	banner?: string
+): Promise<RollupOutput['output']> {
+	const bundle = await rollup( {
+		input,
+		plugins: [
+			splitCss()
+		]
+	} );
+
+	const { output } = await bundle.generate( { format: 'esm', banner } );
+
+	return output;
+}
+
+test( 'Import of single `CSS` file', async () => {
+	const output = await generateBundle( join( import.meta.dirname, './fixtures/single-import/input.ts' ) );
```

Comment:
AFAIK you should be able to use `./fixtures/single-import/body.css` as input, which would allow you to delete the `input.ts` file.

If this works, you can do the same for other tests.

Reply:
@filipsobol Yes, it can be done like you suggest but it will be ok just for two or three testing cases.

From my point of view it can be missunderstood to someone who will look into tests, when we put a `css` file as in input when we default will point to a`ts` file.

Tests that will point to `ts` or `css` wont be groupped by input extension but spreaded across whole test suite.

Just thinking out loud to get all pros and cons for this suggested change.

Maybe we can do a sibgle test to show that we can make an input as a `css` file, WDYT?